lib/semantics/expression.cc

```diff
@@ -1581,47 +1603,25 @@ const Symbol *ExpressionAnalyzer::ResolveGeneric(const Symbol &symbol,
   }
   if (elemental) {
     return elemental;
+  }
+  if (semantics::IsGenericDefinedOp(symbol)) {
+    Say("No specific procedure of generic operator '%s' matches the actual arguments"_err_en_US,
```

Comment:
Maybe `OPERATOR('%s')`?

Reply:
This is consistent with other errors about defined operators but it might be a good idea to change them all. But not part of this PR.